arcsi/static/docs.json

```diff
@@ -0,0 +1,960 @@
+            "id": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "number": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "name": {
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
+            },
+            "language": {
+              "type": "string"
+            },
+            "play_date": {
+              "type": "string",
+              "format": "date"
+            },
+            "image_url": {
+              "type": "string"
+            },
+            "play_file_name": {
+              "type": "string"
+            },
+            "live": {
+              "type": "boolean"
+            },
+            "broadcast": {
+              "type": "boolean"
+            },
+            "airing": {
+              "type": "boolean"
+            },
+            "archive_lahmastore": {
+              "type": "boolean"
+            },
+            "archive_lahmastore_canonical_url": {
+              "type": "string"
+            },
+            "archive_mixcloud": {
+              "type": "boolean"
+            },
+            "archive_mixcloud_canonical_url": {
+              "type": "string"
+            },
+            "archived": {
+              "type": "boolean"
+            },
+            "download_count": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "uploader": {
+              "type": "string"
+            },
+            "shows": {
+              "type": "object",
+              "additionalProperties": {
+                "$ref": "#/components/schemas/show"
+              }
+            }
+          }
+        },
+        "ItemRequests": {
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/components/schemas/ItemRequest"
+          }
+        },
+        "ItemRequestPostBody": {
+          "type": "object",
+          "properties": {
+            "number": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "name": {
+              "type": "string"
+            },
+            "description": {
+              "type": "string"
+            },
+            "language": {
+              "type": "string"
+            },
+            "play_date": {
+              "type": "string",
+              "format": "date"
+            },
+            "live": {
+              "type": "boolean"
+            },
+            "broadcast": {
+              "type": "boolean"
+            },
+            "archive_lahmastore": {
+              "type": "boolean"
+            },
+            "archive_mixcloud": {
+              "type": "boolean"
+            },
+            "uploader": {
+              "type": "string"
+            },
+            "shows": {
+              "type": "integer",
+              "format": "int32"
+            },
+            "show_name": {
```

Comment:
Where did you find this? I think you are right we can remove this

Reply:
I found them in the POST methods of the Show and Item DetailsSchemas
(arcsi/api/show.py line: 146-154, 229-237; arcsi/api/item.py line: 107-110, 290-293)
I tried to remove them but as I saw without them the methods won't work correctly.